### PR TITLE
ops: bring back scripts

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Bring back `ci-builder` to the monorepo as it should have never been deleted. Also bring back various tagging scripts. It is not clear which tagging script is used at the moment, need clarity on which should be actually brought back before merging this. `ci-versions.js` is required for changesets, so bring that back for sure. `update-op-geth.py` is used but unclear about the 2 other shell scripts.

### CODE REVIEW

File: ops/scripts/tag-bedrock-go-modules.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
```
Comment (trianglesphere) [inline ops/scripts/tag-bedrock-go-modules.sh:1]: Not used as far as I know.

File: ops/scripts/update-op-geth.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
```
Comment (trianglesphere) [inline ops/scripts/update-op-geth.py:1]: I believe this may still be used.

**Comment by zhwrd** [comment]
I believe `tag-bedrock-go-modules.sh` is unused, but [ops/scripts/ci-docker-tag-op-stack-release.sh](https://github.com/ethereum-optimism/optimism/pull/6034/files#diff-a6781a83c6d4ad088a7959d923836ca5aeb5288a9b5a1792ff91faa964bbb618) is used.

File: ops/scripts/ci-docker-tag-op-stack-release.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```
Comment (trianglesphere) [inline ops/scripts/ci-docker-tag-op-stack-release.sh:1]: This is used in CI IIRC.

**Review by zhwrd — APPROVED** [review]

**Pull request merged**